rust/sdk-processor/README.md

```diff
@@ -0,0 +1,13 @@
+# Indexer GRPC Parsers (SDK)
+
+Indexer GRPC parser is to indexer data processor that leverages the indexer grpc data. These processors have been (re)-written using the new SDK.
```

Comment:
I'd reword this a bit. Also "Indexer GRPC" is a name used internally. Externally it's known in the docs as "Transaction Stream", and we need to update this everywhere lol 

```suggestion
Processors that index data from the Aptos Transaction Stream (GRPC). These processors have been (re)-written using the new Indexer SDK.
```